src/sentry/static/sentry/app/views/eventsV2/table/tableView.tsx

```diff
@@ -446,4 +458,31 @@ const StyledIcon = styled(IconStack)`
   vertical-align: middle;
 `;
 
+type TopNIndicatorProps = {
+  count: number;
+  index: number;
+};
+
+const TopNIndicator = styled('div')<TopNIndicatorProps>`
+  position: absolute;
+  left: -1px;
+  width: 9px;
+  height: 15px;
+  border-radius: 0 3px 3px 0;
+
+  background-color: ${p => {
+    // this background color needs to match the colors used in
+    // app/components/charts/eventsChart so that the ordering matches
+
+    // the color pallete contains n + 2 colors, so we subtract 2 here
+    return p.theme.charts.getColorPalette(p.count - 2)[p.index];
+  }};
+
+  & span {
```

Comment:
There doesn't seem to be a child span anymore.

Reply:
removed